debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -116,6 +127,8 @@ protected void load(Map<String, Object> lastStoredOffset) {
         this.txId = ((Number) lastStoredOffset.get(TXID_KEY)).longValue();
         this.useconds = (Long) lastStoredOffset.get(TIMESTAMP_KEY);
         this.snapshot = lastStoredOffset.containsKey(SNAPSHOT_KEY);
+        this.schemaName = (String) lastStoredOffset.get(SCHEMA_NAME_KEY);
```

Comment:
I don't think this info should be part of the *offset* as passed to Kafka Connect, only it should be added to the `source` `Struct`. Unfortunately, the class here is mangling the two aspects a bit, so I see how it can be confusing. Also see [DBZ-797](https://issues.jboss.org/browse/DBZ-797) which is about a similar case, i.e. a field is added to the "source" block although it's only meant for the offset.

The design is clearer for the new Oracle and SQL Server connectors, and eventually we should do the same for Postgres and the other existing connectors. No need to do this now as part of this change here, though, just let's not add more feels to the offset that are not needed there.